v3.0.0-rc.4、v3.0.0-rc.5 ，使用 require.context 加载指定目录模块时编译失败

### 问题描述

Taro v3.0.0-rc.4、Taro v3.0.0-rc.5 ，使用 require.context 加载模块时编译失败。
### 复现步骤

<!-- 复现问题的步骤。代码只贴截图，不贴文字会被视为无效issue -->

1. 无论 Taro v3.0.0-rc.4、Taro v3.0.0-rc.5 都会有这个问题，Taro v3.0.0-beta.6 正常
2. 使用 require.context 指定目录下的所有模块
```js
var configResolers = require.context("./", true, /config\.\w+\.js$/);
```

### 期望行为

可以正常使用 `require.context`

### 报错信息

```shell
🙅   编译失败.

./src/config/index.js
Module not found: Can't resolve './' in '/Users/Edwin/codes/taro/src/config'

./src/config/index.js
Module not found: Error: Can't resolve './' in '/Users/Edwin/codes/taro/src/config'
resolve './' in '/Users/Edwin/codes/taro/src/config'
  Parsed request is a directory
  using description file: /Users/Edwin/codes/taro/package.json (relative path: ./src/config)
    resolve multi platform file path
      using description file: /Users/Edwin/codes/taro/package.json (relative path: ./src/config)
        using description file: /Users/Edwin/codes/taro/package.json (relative path: ./src/config/index.js)
          as directory
            /Users/Edwin/codes/taro/src/config/index.js is not a directory
```
### 系统信息

```shell
  Taro CLI 3.0.0-rc.5 environment info:
    System:
      OS: macOS 10.14.6
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 12.13.1 - ~/.nvm/versions/node/v12.13.1/bin/node
      npm: 6.12.1 - ~/.nvm/versions/node/v12.13.1/bin/npm
    npmPackages:
      @tarojs/cli: 3.0.0-rc.5 => 3.0.0-rc.5
      @tarojs/components: 3.0.0-rc.5 => 3.0.0-rc.5
      @tarojs/taro: 3.0.0-rc.5 => 3.0.0-rc.5
      @tarojs/webpack-runner: 3.0.0-rc.5 => 3.0.0-rc.5
      eslint-config-taro: 3.0.0-rc.5 => 3.0.0-rc.5
      react: 16.13.1 => 16.13.1
```

### 补充信息

看看 `MultiPlatformPlugin` 及关联上下游的解析逻辑

--- a/packages/taro-runner-utils/src/resolve/MultiPlatformPlugin.ts
+++ b/packages/taro-runner-utils/src/resolve/MultiPlatformPlugin.ts
@@ -37,7 +37,8 @@ export class MultiPlatformPlugin {
       .getHook(this.source)
       .tapAsync('MultiPlatformPlugin', (request, resolveContext, callback) => {
         const innerRequest: string = request.request || request.path
-        if (!innerRequest) return callback()
+        // request.directory will be true when is using require.context
+        if (!innerRequest || request.directory) return callback()
 
         if (!path.extname(innerRequest)) {
           let srcRequest: string

